CorsixTH/Lua/game_ui.lua

```diff
@@ -117,6 +117,24 @@ function GameUI:makeVisibleDiamond(scr_w, scr_h)
   }
 end
 
+--! Calculate the minimum valid zoom value
+--!
+--! Zooming out too much would cause negative width/height to be returned from
+--! makeVisibleDiamond. This function calculates the minimum zoom_factor that
+--! would be allowed.
+function GameUI:calculateMinimumZoom()
```

Comment:
Add your tolerance directly to the MinimumZoom returned by the function, since it's that value that is off, not the usage of it.

I myself don't see a way around it. Be sure to document the magic number, and please include a space around your operator.